include/astshim/PolyMap.h

```diff
@@ -52,6 +52,9 @@ friend class Object;
     /**
     Construct a @ref PolyMap with specified forward and inverse transforms.
 
+    The two sets of coefficients are independent of each other: the inverse transform
+    need not undo the forward transform.
+
     @param[in] coeff_f  A `(2 + nin) x ncoeff_f` matrix of coefficients.
```

Comment:
Something that came up when working on DM-9598: should this be `(2 + nin) x ncoeff_f`, or `ncoeff_f x (2 + nin)`? When making an `ndarray::Array` to pass here it appears you need to give the dimensions in the latter order.